Dockerfile

```diff
@@ -0,0 +1,44 @@
+# Based on the official Node.js 0.10 image, 
+# because it's the easiest way to get that version and the image is small enough
+FROM node:0.10-slim
+
+# Put our Node.js app definition in place
+COPY package.json /opt/nfs/
+
+# Install our Node.js app
+# $buildDeps will be added for the `npm install`, then removed immediately after
+# the resulting image layer will not include the size of the $buildDeps
+RUN \
+    buildDeps='git g++ make python' \
+    runDeps='nfs-common ca-certificates curl' \
+    && set -x \
+    && apt-get update && apt-get install -y $buildDeps $runDeps --no-install-recommends \
+    && cd /opt/nfs \
+    && npm install \
+    && rm -rf /var/lib/apt/lists/* \
+    && apt-get purge -y --auto-remove $buildDeps
+
+# Add Containerbuddy and its configuration
+ENV CONTAINERBUDDY_VER 1.3.0
+ENV CONTAINERBUDDY_CHECKSUM c25d3af30a822f7178b671007dcd013998d9fae1
+ENV CONTAINERBUDDY file:///etc/containerbuddy.json
+
+RUN export CB_SHA1=c25d3af30a822f7178b671007dcd013998d9fae1 \
```

Comment:
Oh it looks like we're have it repeated in the Jenkins blueprint too... it's both `CONTAINERBUDDY_CHECKSUM` and `CB_SHA1`. I don't see any reason to have both. I think keeping the checksum in the environment after the installation (via the `ENV` stanza) doesn't make a whole lot of sense because it's the checksum of the zip file, which we toss out when we're done.